x/concentrated-liquidity/incentives_test.go

```diff
@@ -3509,3 +3511,241 @@ func (s *KeeperTestSuite) TestPrepareBalancerPoolAsFullRange() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestClaimAndResetFullRangeBalancerPool() {
+	lockableDurations := s.App.PoolIncentivesKeeper.GetLockableDurations(s.Ctx)
+	longestLockableDuration := lockableDurations[len(lockableDurations)-1]
+	uptimeHelper := getExpectedUptimes()
+
+	tests := map[string]struct {
+		existingClLiquidity sdk.Coins
```

Comment:
This is a general suggestion. Let's try to avoid referencing to CL as "CL" anywhere in the codebase please. Instead, expand the abbreviation. I bet anyone new to the codebase would have to spend cycles trying to figure out what various abbreviations mean in a couple of months

```suggestion
		existingConcentratedLiquidity sdk.Coins
```

Reply:
There's an interesting tradeoff here given how verbose "ConcentratedLiquidity" is, but I generally agree. Will adjust in this PR and keep in mind in the future